R/calendar_range.R

```diff
@@ -0,0 +1,114 @@
+#' Create Semantic UI Calendar Range
+#'
+#' This is a wrapper for creating calendar ranges using Semantic UI calendars.
+#' It creates a form composed of two calendars. The selected range values are
+#' available under \code{input[[input_id]]}.
+#'
+#' @details The Semantic UI calendar range automatically pops up the end range
```

Comment:
I would suggest changing 'end range calendar' to 'range end calendar' or 'end date calendar'